src/compiler/compiler.rs

```diff
@@ -83,7 +82,10 @@ impl<'a> Compiler<'a> {
 
         let (errors, warnings): (Vec<_>, Vec<_>) =
             compiler.diagnostics.into_iter().partition(|diagnostic| {
-                matches!(diagnostic.severity(), Severity::Bug | Severity::Error)
+                matches!(
+                    diagnostic.severity(),
+                    crate::diagnostic::Severity::Bug | crate::diagnostic::Severity::Error
```

Comment:
🗒️ The `Optimize Imports` IDE function has failed me here. As always, it doesn't work well with macros. This is a workaround.